ArchUnitNETTests/Fluent/Syntax/Elements/ObjectsShouldTests.cs

```diff
@@ -0,0 +1,3229 @@
+        should.CallAny("^.*::CalledMethod\\(\\)$", true).AssertNoViolations(helper);
+        should.CallAny([helper.CalledMethod.FullName]).AssertNoViolations(helper);
+        should.CallAny(["^.*::CalledMethod\\(\\)$"], true).AssertNoViolations(helper);
+        should.CallAny(helper.CalledMethod).AssertNoViolations(helper);
+        should.CallAny([helper.CalledMethod]).AssertNoViolations(helper);
+        should.CallAny(MethodMembers().That().Are(helper.CalledMethod)).AssertNoViolations(helper);
+
+        helper.AddSnapshotHeader("Violations");
+        should = MethodMembers().That().Are(helper.MethodWithSingleDependency).Should();
+        should.CallAny(helper.MethodWithoutDependencies.FullName).AssertOnlyViolations(helper);
+        should.CallAny([helper.MethodWithoutDependencies.FullName]).AssertOnlyViolations(helper);
+        should.CallAny(helper.MethodWithoutDependencies).AssertOnlyViolations(helper);
+        should.CallAny([helper.MethodWithoutDependencies]).AssertOnlyViolations(helper);
+        should
+            .CallAny(MethodMembers().That().Are(helper.MethodWithoutDependencies))
+            .AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Non-existent method member");
+        should = MethodMembers().That().Are(helper.MethodWithSingleDependency).Should();
+        should.CallAny(helper.NonExistentObjectName).AssertOnlyViolations(helper);
+        should.CallAny([helper.NonExistentObjectName]).AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Empty arguments");
+        should = MethodMembers().That().Are(helper.MethodWithSingleDependency).Should();
+        should.CallAny(new List<string>()).AssertOnlyViolations(helper);
+        should.CallAny(new List<MethodMember>()).AssertOnlyViolations(helper);
+        should
+            .CallAny(MethodMembers().That().Are(helper.NonExistentObjectName))
+            .AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Multiple arguments");
+        should = MethodMembers().That().Are(helper.MethodWithMultipleDependencies).Should();
+        should
+            .CallAny(
+                [
+                    helper.MethodWithoutDependencies.FullName,
+                    helper.MethodWithMultipleDependencies.FullName
+                ]
+            )
+            .AssertOnlyViolations(helper);
+        should
+            .CallAny(helper.MethodWithoutDependencies, helper.MethodWithMultipleDependencies)
+            .AssertOnlyViolations(helper);
+        should
+            .CallAny([helper.MethodWithoutDependencies, helper.MethodWithMultipleDependencies])
+            .AssertOnlyViolations(helper);
+        should
+            .CallAny(
+                MethodMembers()
+                    .That()
+                    .Are(helper.MethodWithoutDependencies, helper.MethodWithMultipleDependencies)
+            )
+            .AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Multiple inputs");
```

Comment:
This header seems to not fit as there is only one input each in the following two tests.